### DISCUSSION TITLE
Remove the () in address book page

### DISCUSSION THREAD

**Discussion by kjincgroup** [BitCannaGlobal/WeedMobile#85] [General]
![address_book_page](https://github.com/BitCannaGlobal/WeedMobile/assets/111372349/00236a16-502a-41e0-be08-586cc2a2a9df)

It is weird to have a () next to the contact name

**Comment by TJ-BitCanna** [comment]
Thanks for notifying!

If you would create an address with a standard memo it is shown between the brackets.

But maybe it should have a condition;
IF memo exists, then show (memo)
IF no memo exists, then don't show ()

**Comment by kjincgroup** [comment]
I can confirm the issue is fixed. Thank you team!
